webui/react/src/components/CheckpointModalTrigger.tsx

```diff
@@ -31,20 +40,6 @@ const CheckpointModalTrigger: React.FC<Props> = ({
   const { contextHolder: modalModelCreateContextHolder, modalOpen: openModalCreateModel } =
     useModalModelCreate({ onClose: handleOnCloseCreateModel });
 
-  const handleOnCloseCheckpointRegister = useCallback(
-    (reason?: ModalCloseReason, checkpoints?: string[]) => {
-      if (checkpoints) openModalCreateModel({ checkpoints });
-    },
-    [openModalCreateModel],
-  );
-
-  // Has to use var to hoist openModalCheckpointRegister for use above
-  /* eslint-disable-next-line no-var */
-  var {
```

Comment:
Using `var` is not ideal and in the new TS version. TS compiler complains this kind of code with `Variable 'openModalCheckpointRegister' is used before being assigned.`, so I reorganized it